jobs/rocoto/ecenfsoi.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -x
```

Comment:
All of these should be `#! /usr/bin/env bash` and be followed by `source "$HOMEgfs/ush/preamble.sh"` as the first command.